drivers/sht3x/sht3x.c

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_sht3x
+ * @brief       Device Driver for Sensirion SHT30/SHT31/SHT35 Humidity and Temperature Sensors
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ */
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+#include "log.h"
+
+#include <errno.h>
+#include <string.h>
+
+#include "sht3x.h"
+#include "xtimer.h"
+
+#define ASSERT_PARAM(cond) \
+    if (!(cond)) { \
+        DEBUG("[sht3x] %s: %s\n", \
+              __func__, "parameter condition (" #cond ") not fulfilled"); \
+        assert(cond); \
+    }
+
+#define DEBUG_DEV(f, d, ...) \
+        DEBUG("[sht3x] %s dev=%d addr=%02x: " f "\n", \
+              __func__, d->i2c_dev, d->i2c_addr, ## __VA_ARGS__);
+
+#define ERROR_DEV(f, d, ...) \
+        LOG_ERROR("[sht3x] dev=%d addr=%x: " f "\n", \
+                  d->i2c_dev, d->i2c_addr, ## __VA_ARGS__);
+
+/** SHT3x common commands */
+#define SHT3X_STATUS_CMD               0xF32D
```

Comment:
@maribu 

> Maybe you could also improve the documentation a bit by grouping the commands:

Hm ... C files are not included in the doxygen documentation. So I do not see the need to group and comment them in a doxygen-compliant way. Furthermore, since these commands are only used internally, I see no reason to make them available in a header file. In addition, the names are self-explanatory, so I don't think that it become more informative by adding a comment per command.

Reply:
You're right. I agree that using Doxygen syntax when Doxygen is not actually including the files has no have. My bad.